### PR TITLE
Add blog page and draft tip of the week blog post

### PR DESCRIPTION
**Pull request by d33bs**
Added blog page and draft tip of the week blog post covering software linting.

Some notes:
- I noticed that code blocks were being center-aligned by default and had inaccessible color contrast so made adjustments to `code.scss`.
- Maybe we should remove the "tip-of-the-week" tag and titling to fall back on it being a "just" a blog post instead?

### CODE REVIEW

File: css/code.scss
Comment (falquaddoomi) [inline css/code.scss:30]: Nice! These changes both make sense, but I wonder if `code.highlighter-rouge`'s background (a few lines above) should also be changed to be this div's color. It could also be vice versa, as I find the `code.highlighter-rouge`'s background sufficiently dark to read it.

Comment (falquaddoomi) [reply css/code.scss:30]: While we're tweaking the appearance of code blocks, it might be nice to include a bit of rounding on the boxes, e.g. `border-radius: 3px`. I don't know how that plays into the visual language of the site, so I'll leave it to @vincerubinetti to approve that or not.

**Review by falquaddoomi — APPROVED** [review]
Nice kickoff to the blog; thanks for setting it up! The content all looks good, too. I had a minor consistency nitpick which IMHO isn't blocking merging, so I'll approve for now.

File: css/code.scss
Comment (vincerubinetti) [reply css/code.scss:30]: I will add a $rounded variable to the upstream lab website template for users who want that.

**Review by vincerubinetti — APPROVED** [review]

File: css/code.scss
Comment (d33bs) [reply css/code.scss]: Thank you @vincerubinetti and @falquaddoomi !

**Pull request merged**
